### PR TITLE
Update the game over message

### PR DESCRIPTION
**Pull request by OppenHaix**
Update the game over message so people know how to play again!

### CODE REVIEW

**Review by OppenHaix — COMMENTED** [review]
Update the game over message so people know how to play again!

File: README.md [hunk 1]
```diff
@@ -14,51 +14,30 @@ _Collaborate and work together on GitHub._
 </header>
 
 <!--
-  <<< Author notes: Course start >>>
```
Comment (starxicn) [inline README.md:17]: Update the game over message

File: README.md [hunk 2]
```diff
@@ -14,51 +14,23 @@ _Collaborate and work together on GitHub._
...
-  Include start button, a note about Actions minutes,
-  and tell the learner why they should take the course.
+  <<< Author notes: Step 5 >>>
+  Start this step by acknowledging the previous step.
+  Define terms and link to docs.github.com.
 -->
 
-## Welcome
+## Step 5: Apply suggested changes
```
Comment (starxicn) [inline README.md:22]: Update the game over message

Comment (starxicn) [reply README.md:22]: Update the game over message

**Comment by OppenHaix** [comment]
Update the game over message so people know how to play again!

File: README.md [hunk 3]
```diff
@@ -14,51 +14,23 @@ _Collaborate and work together on GitHub._
...
 
-All great projects start with collaboration. Pull requests are the foundation of teamwork on GitHub — and pull request reviews give you the ability to work together and discuss changes specific to a pull request by commenting, requesting changes, or approving.
+_Nicely done suggesting changes! :partying_face:_
 
-- **Who is this for**: Developers, new GitHub users, users new to Git, students, managers, teams.
-- **What you'll learn**: When and how to request a review; how to provide a review of someone else's pull request.
-- **What you'll build**: We'll be reviewing a pull request for a simple game.
-- **Prerequisites**: We assume you are familiar with creating branches, commits, and pull requests—you can learn this in our [Introduction to GitHub](https://github.com/skills/introduction-to-github) course.
-- **How long**: This course takes less than 30 minutes to complete.
```
Comment (starxicn) [inline README.md:30]: Update the game over message so people know how to play again!

File: index.html [hunk 4]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
```
Comment (starxicn) [inline index.html:50]: 好好好

File: README.md (hunk 3 above)
Comment (starxicn) [reply README.md:30]: 11111

File: index.html (hunk 4 above)
Comment (starxicn) [reply index.html:50]: 1111

**Review by OppenHaix — COMMENTED** [review]
Update the game over message

File: index.html [hunk 5]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
...
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```
Comment (starxicn) [inline index.html:50]:
```suggestion
<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```
Update the game over message so people know how to play again!

Comment (OppenHaix) [inline index.html]:
```suggestion
<h2 hidden>Game over, refresh to play again !</h2>
```
Update the game over message

**Pull request merged**